site/kube-addon.md

```diff
@@ -62,6 +62,32 @@ Shut down Kubernetes, and _on all nodes_ perform the following:
 Then relaunch Kubernetes and install the addon as described
 above.
 
+**Note:** You can customise the generated YAML file by passing Weave Net's options, arguments and environment variables as query parameters:
```

Comment:
Had missed this after my previous change, as GitHub had hidden it. Will change then :+1: